cstar/system/environment.py

```diff
@@ -318,9 +303,13 @@ def load_lmod_modules(self, lmod_file) -> None:
         for mod in lmod_list:
             self._call_lmod(f"load {mod}")
 
-    def set_env_var(self, key: str, value: str) -> None:
-        """Set value of an environment variable and store it in the user environment
-        file.
+    @staticmethod
```

Comment:
Per your comment, this method was created to wrap the need to update both the dotenv file and local environment. Can we just add a `# TODO: remove method, user-env updates are deprecated and this is no longer necessary` instead of the story? `# TODO` will be easy to find later, versus the blurb.